src/pages/login.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bella Coola</title>
+    <link rel="stylesheet" href="../../styles.css" />
+  </head>
+  <body>
+    <div class="app-outer-layout">
+      <div class="app-inner-layout login">
+        <div class="login-topside">
+          <img src="../assets/icons/app-large-icon.png" alt="icon" />
+          <div class="login-buttons">
+            <button id="login">Login</button>
+            <button id="signup">Sign-up</button>
+          </div>
+        </div>
+        <div class="login-signup-menu">
+          <form class="login-form">
+            <div class="email-container">
+              <label for="email"
+                >Email
+                <address></address
```

Comment:
what is the purpose of this empty tag?

Reply:
@docdrue there is no purpose. I'll make sure to delete it.